src/main/java/seedu/address/model/Model.java

```diff
@@ -115,4 +115,4 @@
         String getName();
     }
 
-}
+}
```

Comment:
@nightism already mentioned this in your previous PR -- there needs to be a newline here.